gaztec/src/components/sidebar/components/createAccountDialog.tsx

```diff
@@ -0,0 +1,88 @@
+import DialogTitle from "@mui/material/DialogTitle";
+import Dialog from "@mui/material/Dialog";
+import { AccountWalletWithSecretKey, Fr, PXE } from "@aztec/aztec.js";
+import { getSchnorrAccount } from "@aztec/accounts/schnorr";
```

Comment:
This as a separate import seems fine. I'd imagine wallets would manage their own account contracts